### PR TITLE
For whole-doc exports, calculate top/left transparent padding to acco…

### PR DESCRIPTION
**Pull request by mcilroyc**
…unt for Photoshop trimming to visible pixels.

Addresses PS-12498

### CODE REVIEW

File: lib/renderer.js [hunk 1]
```diff
@@ -892,6 +893,12 @@
                 inputRect = layer.bounds;
                 paddedBounds = inputRect;
             }
+        } else if (component.layerId === 0) { // whole doc
+            inputRect = component.document.bounds;
+            paddedBounds = component.document.bounds;
+
+            var contentBounds = this._getContentBounds(component.document.layers, false, component.document.bounds);
```
Comment (samgannawayA) [inline lib/renderer.js]: For whole doc, why calculate ContentBounds? Isn't it just the doc bounds?

Comment (mcilroyc) [reply lib/renderer.js]: Added code comments to explain

Comment (samgannawayA) [reply lib/renderer.js]: We're getting contentBounds so we can be prepared for the state of the pixmap we will receive, right?

Comment (mcilroyc) [reply lib/renderer.js]: correct

**Pull request merged**